docker-storage-setup.sh

```diff
@@ -931,46 +938,81 @@ create_lv() {
 return 0
 }
 
-setup_docker_root_volume() {
-  if [ -z "$DOCKER_ROOT_VOLUME_SIZE" ]; then
-    Fatal "Specify a valid value for DOCKER_ROOT_VOLUME_SIZE."
+setup_extra_volume() {
+  local lv_name=$1
+  local mount_dir=$2
+  local lv_size=$3
+
+  if [ -z "$lv_size" ]; then
+    Fatal "Specify a valid value for CONTAINER_ROOT_LV_SIZE."
   fi
 
-  if ! check_data_size_syntax $DOCKER_ROOT_VOLUME_SIZE; then
-    Fatal "DOCKER_ROOT_VOLUME_SIZE value $DOCKER_ROOT_VOLUME_SIZE is invalid."
+  if ! check_data_size_syntax $lv_size; then
+    Fatal "CONTAINER_ROOT_LV_SIZE value $lv_size is invalid."
```

Comment:
same here. we should move this check in to caller and then change message appropriately.